### ISSUE TITLE
Compilation errors in bio_dgram_test on master

### ISSUE THREAD

**Issue by mattcaswell** [openssl/openssl#19349]
bio_dgram_test is failing to compile in some builds on master at the moment. For example see:
https://github.com/openssl/openssl/actions/runs/3190491507/jobs/5205668303

````
test/bio_dgram_test.c:498:24: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
    BIO_MSG msgs[2] = {0}, rmsgs[2] = {0};
                       ^
                       {}
test/bio_dgram_test.c:498:40: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
    BIO_MSG msgs[2] = {0}, rmsgs[2] = {0};
                                       ^
                                       {}
2 errors generated.
make[1]: *** [Makefile:17474: test/bio_dgram_test-bin-bio_dgram_test.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [Makefile:280[5](https://github.com/openssl/openssl/actions/runs/3190491507/jobs/5205668303#step:6:6): build_sw] Error 2
Error: Process completed with exit code 2.
````

**Comment by t8m** [comment]
Fixed by merging #19311

**Comment by wan7teen-ping** [comment]
Hi I will help you with everything you need but you have to subscribe to
the youtube channel and I will fulfill all your requests

https://youtu.be/aIaqr4eq18g

Pada Rab, 5 Okt 2022, 11:16 PM Matt Caswell ***@***.***>
menulis:
